pages/dev/mobile/_postId.vue

```diff
@@ -0,0 +1,180 @@
+<template>
```

Comment:
Not sure if these are meant to be used or are just WIP, but they're broken right now with "window is not defined" error. If still WIP, then please document with a few comments for ease of coming back to it